plugins/gatsby-plugin-generate-doc-json/utils/handlers.js

```diff
@@ -13,23 +17,50 @@ const getAllAttributes = (node) =>
       }, {})
     : {};
 
+const getSrcUrl = (fileRelativePath, url) =>
+  fileRelativePath
+    .replace('.mdx', '')
+    .replace('src/content/', '')
+    .split('/')
+    .slice(0, -1)
+    .join('/')
+    .concat(url.replace('./', '/'));
```

Comment:
Is this trying to get the full relative path to the image given the MDX file its in? You can simplify this with `path` helpers:

```js
path.join(path.dirname(fileRelativePath.replace('src/content', '')), url)
```